roles/openshift_logging/tasks/oc_secret.yaml

```diff
@@ -0,0 +1,7 @@
+---
+- command: >
```

Comment:
@jcantrill  looks like `oc_secret` was merged. can we try to use that module instead of this task?